src/components/base/JInputField/jInputField.m.scss

```diff
@@ -15,12 +15,7 @@
 .wrap {
   height: 72px;
   padding: 0;
-  box-shadow: $normal-shadow;
```

Comment:
All inputs have shadow in design. Why did you remove it?

Reply:
This was not removed, it was moved to the color theme section, because we need to hide shadow in the disabled state
```
.white,
.white-indicator {
  .wrap {
    background-color: $white;	    background-color: $white;
    box-shadow: $normal-shadow;

     &:hover {
      box-shadow: $hover-shadow;
    }
  }
...
```